roles/kube_prometheus/hack/fix-dashboard.py

```diff
@@ -66,6 +66,14 @@ def main(dashboard_data, output, filename, title=None):
                 annotation['datasource'] = source_template
                 change = True
 
+    sys.stderr.write("{}\n".format(tags))
+    if tags is not None:
+        # reset initial tags list
+        dashboard['tags'] = []
```

Comment:
I'd still write the one-line comprehension from #249 but sure, ok :wink:

Reply:
mm i think i miss a commit, i made the one line as you suggested.